modules/vsphere-multiple-hosts.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/vsphere-hosts.adoc
+
+[id="vsphere-multiple-hosts_{context}"]
+= Configuring multiple vSphere hosts
+
+You can configure your cluster to run on multiple VMware vSphere hosts.
+
+.Prerequisites
+
+* You installed a {product-title} cluster on vSphere.
+* You reviewed the
+link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html[vSphere Cloud Provider Configuration]
+and can provide the data that is required for the `VirtualCenter` hosts that you
+add.
+
+.Procedure
+
+. Edit the `cloud-provider-config` ConfigMap:
++
+----
+$ oc edit configmap/cloud-provider-config
```

Comment:
This needs a `-n openshift-config` to get the configmap from the openshift-config namespace.